### PR TITLE
Cloud metafiles config succeed

### PR DESCRIPTION
**Pull request by LinerSu**

### CODE REVIEW

**Comment by alex-s-hong** [comment]
Can we still use PostgreSQL for our cloud service instead of cloudant?

File: manifest.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  memory: 64M
+  #random-route: true
+  routes:
+  - route: nyu-product-service-f19.mybluemix.net
+  disk_quota: 1024M
+  buildpack: python_buildpack
+  timeout: 180
+  #command: gunicorn --bind=0.0.0.0:$PORT service:app
+  services:
+  - PostgreSQL
```
Comment (LinerSu) [inline manifest.yml:16]: I already changed services as `PostgreSQL`

Comment (LinerSu) [reply manifest.yml:16]: @alex-s-hong

**Comment by ianchou19** [comment]
> Can we still use PostgreSQL for our cloud service instead of cloudant?

Yes, IBM Cloud do have PostgreSQL service.

**Pull request merged**